dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java

```diff
@@ -1,102 +1,166 @@
+
+        //update contentlets
+        query.append("update identifier ident,\n")
+                .append("(select distinct temp.identifier myID, temp.mod_user, inode.idate, ")
+                .append("struc.velocity_var_name  from contentlet temp, inode, structure struc,\n")
+                .append("(select identifier, MIN(idate) as idate from contentlet, inode where ")
+                .append("inode.inode=contentlet.inode group by identifier) custom_select\n")
+                .append("where temp.identifier=custom_select.identifier\n")
+                .append("and inode.inode=temp.inode and inode.idate=custom_select.idate \n")
+                .append("and temp.structure_inode = struc.inode) my_query\n")
+                .append("set ident.owner=my_query.mod_user, ident.create_date=my_query.idate\n")
+                .append("where ident.id=my_query.myID;");
+
+        return query.toString();
+    }
+
+    @Override
+    public String getOracleScript() {
+
+        final StringBuilder query = new StringBuilder();
+
+        //update templates
+        query.append(getQueryToUpdateNonContentletsOracle("template"));
+
+        //update containers
+        query.append(getQueryToUpdateNonContentletsOracle("dot_containers"));
+
+        //update links
+        query.append(getQueryToUpdateNonContentletsOracle("links"));
+
+        //update folders
+        query.append(getQueryToUpdateNonContentletsOracle("folder"));
+
+        //update contentlets
+        query.append("MERGE INTO identifier ident\n")
+                .append("    USING\n")
+                .append("(select distinct temp.identifier myID, temp.mod_user, inode.idate, struc.velocity_var_name  from contentlet temp, inode, structure struc,\n")
+                .append("(select identifier, MIN(idate) as idate from contentlet, inode where inode.inode=contentlet.inode group by identifier) custom_select\n")
+                .append("where temp.identifier=custom_select.identifier\n")
+                .append("and inode.inode=temp.inode and inode.idate=custom_select.idate \n")
+                .append("and temp.structure_inode = struc.inode)\n")
+                .append("st ON (ident.id = st.myID) \n");
+
+        return query.toString();
+    }
+
+    @Override
+    public String getMSSQLScript() {
+        return getPostgresScript();
+    }
+
+    @Override
+    public String getH2Script() {
+        return null;
+    }
+
+    @Override
+    protected List<String> getTablesToDropConstraints() {
+        return null;
+    }
+
+    private String getQueryToUpdateNonContentletsOracle(final String tableName){
+        final StringBuilder query = new StringBuilder();
+        return query.append("MERGE INTO identifier ident\n")
+                .append("    USING\n")
+                .append("(select distinct temp.identifier myID, owner iowner, inode.idate idate from ")
+                .append(tableName).append(" temp, inode, \n")
```

Comment:
Codacy found an issue: [StringBuffer (or StringBuilder).append is called 2 consecutive times with literals. Use a single append with a single combined String.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6393742)